src/models/serialportctrl.ts

```diff
@@ -0,0 +1,182 @@
+import * as vscode from 'vscode';
+import * as _ from 'lodash';

```

Comment:
Avoid importing the entire `lodash`, it's very very huge. Use its member which you need instead

Reply:
Fixed.